bc_obps/registration/models/operation.py

```diff
@@ -47,6 +47,7 @@ class Statuses(models.TextChoices):
         db_comment="The operator who owns the operation",
         related_name="operations",
     )
+    # brianna do we need this? can't we find out by looking at the foreign keys in the multiple operator table?
     operation_has_multiple_operators = models.BooleanField(
```

Comment:
I think we should be getting this from the foreign keys in the multiple operator table so we don't have to set it separately (and potentially introduced conflicting sources of truth). The multiple op models were never used in reg 1 prod, but this table is, so is it changeable at this point?